nle/env/base.py

```diff
@@ -312,15 +310,15 @@ def __init__(
         else:
             ttyrec = None
 
-        self.env = nethack.Nethack(
+        self.nethack = nethack.Nethack(
```

Comment:
this is a much better renaming, though im sure will annoy downstream users.

Reply:
so funny how easy it is to ignore "call it what it is!"

Reply:
That's true re: downstream users. Hence I at least included @samvelyan here.

Reply:
We could include an alias.

Then again, this is easy to fix.

Reply:
Thanks for flagging this @heiner. I'll incorporate this into MiniHack after NLE's next release.

Reply:
Again, if it's easier we can add an alias.

Reply:
Hi team, do you know when there will be the next release of MiniHack? I'm trying to use recent nle because of issue fixed by https://github.com/facebookresearch/nle/pull/299